portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -97,6 +97,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-170577 - Verify toast alert with action button disappears after 10 seconds"
+	@priority = "5"
+	test DisappearsAfter10SecsWithActions {
+		task ("When a toast alert appears") {
+			Click(
+				key_text = "Disappears After 10 Seconds",
+				locator1 = "Button#ANY");
+		}
+
+		task ("And there is an action button on it") {
+			VerifyElementPresent(
+				key_tag = "//a",
```

Comment:
Make this xpath more specific. For example, use `//*[@id='ToastAlertContainer']` to clarify that the link is on the alert. This will make the test more robust and not prone to mistakenly target a link on the page from future changes or artifacts from previous tests.

Reply:
Also, create a new path in ClayAlertToast.path for it.

Reply:
I tried to add `//*[@id='ToastAlertContainer']` to the toast alert but as I created it based on [Clay alert documentation](https://clayui.com/docs/components/alert.html) and having in mind that the openToast function is structured differently on some aspects, I didn't know how to include this id. @matuzalemsteles could you give some light on what should I do to make this alert have `//*[@id='ToastAlertContainer']` as the other alerts that use openToast function?

Reply:
@deborasoliveira in the `openToast` method call you can pass the `containerId: 'ToastAlertContainer'` property to add an `id` to the toast container. Let me know if it would be expected.

Reply:
Hmm I see, you can pass the `id` property in the `ClayAlert.ToastContainer` component to add the id to the container.

Reply:
thanks @matuzalemsteles! I did it as you said and it worked 🎉